libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.ts

```diff
@@ -248,6 +258,7 @@ export class ModalWrapperComponent implements Modal, AfterViewInit, OnInit, OnDe
   private listenForIonModalWillDismiss() {
     if (this.ionModalElement) {
       this.ionModalElement.addEventListener('ionModalWillDismiss', () => {
+        this.renderer.removeClass(document.body, 'allow-scroll');
```

Comment:
Please use `windowRef` instead:
```suggestion
        this.renderer.removeClass(this.windowRef.document.body, 'allow-scroll');
```

Reply:
Fixed in next commit